dateutil/test/test_isoparser.py

```diff
@@ -55,7 +55,7 @@ def test_year_only(dt):
 DATES += [datetime(2000, 2, 1), datetime(2017, 4, 1)]
 @pytest.mark.parametrize('dt', tuple(DATES))
 @pytest.mark.parametrize('fmt',
-    ['%Y%m', '%Y-%m'])
+    ['%Y-%m', '%Y-%m'])
```

Comment:
I think this just repeats the same test twice. Since there's only one valid year-month format, I think you can actually move `fmt` into the function definition.

Reply:
Done.